frontend/public/favicon-16x16.png

```diff
@@ -0,0 +1 @@
+/home/xitij/projects/opencraft/ocim/static/img/favicon/favicon-16x16.png
```

Comment:
Seems symbolic links were committed instead of the files.

Reply:
That is on purpose, to avoid duplication. I will fix it to be relative though.